README.md

```diff
@@ -75,6 +76,30 @@ few things:
   have the new prefix except those matching the `*-mode*` pattern,
   which will retain the `org-roam-bibtex` namespace.**
 
+<a name="orb-help-me"></a>HELP ME!!!
+---------------
+
+Please read carefully this section before installing Org Roam BibTeX.  There
+are good chances you'll experience different problems and issues when running
+or even installing Org Roam BibTeX.  These chances primarily correlate with
+your level of comfort with Emacs, Lisp, Org mode, Org Roam, BibTeX and other
+such things.  There may also be bugs that obviously do not depend on your
+skills.  We hunt them, eliminate them, then introduce new ones, and so on.  In
+any case, it's very probable that you'll need some help.  We are glad to help
+you, sure, and kindly ask you to follow these simple rules when asking for
+help:
+
+1. Read carefully this README file.
+2. Read it once again, please.
+3. Check our [community resources](#community) and describe your problem there.
+4. "Help me!!1!11!!!" is wrong. "Help me, please" is correct. Anywhere. Anytime.
```

Comment:
I'm a bit iffy about this one.  I know that we've gotten some of those over the months, but I don't think it bears mentioning, especially if they've gone out of their way to read this section.

Reply:
I meant 4.

Reply:
You're right. It is obvious for the majority.